<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Database_In_Out</class>
 <widget class="QDialog" name="Database_In_Out">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>导出|导入</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res_static.qrc">
    <normaloff>:/icon/source/res/photo/icon/title.png</normaloff>:/icon/source/res/photo/icon/title.png</iconset>
  </property>
  <widget class="QLabel" name="lab_filepath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>导出|导入(源)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Microsoft YaHei UI&quot;;
color:rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbtn_path">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_filetype">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>导出目的格式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_mode">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>158</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>mode</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_filetype_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>开启多线程</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdb_open">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>140</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>开启</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdb_close">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>101</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_close">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font-family:Microsoft Yahei;
	color:white;
	background-color:rgb(14 , 150 , 254);
	border-radius:5px;
}

QPushButton:hover
{
	background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
}
</string>
      </property>
      <property name="text">
       <string>退出程序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_back">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font-family:Microsoft Yahei;
	color:white;
	background-color:rgb(14 , 150 , 254);
	border-radius:5px;
}

QPushButton:hover
{
	background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
}
</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_ok">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font-family:Microsoft Yahei;
	color:white;
	background-color:rgb(14 , 150 , 254);
	border-radius:5px;
}

QPushButton:hover
{
	background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
}
</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lab_filepath_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>导出|导入(目标)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbtn_path_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>90</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Microsoft YaHei UI&quot;;
color:rgb(0, 0, 0);
</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res_static.qrc"/>
 </resources>
 <connections/>
</ui>
